sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_receiver_async.py

```diff
@@ -69,6 +71,7 @@ async def _incoming_transfer(self, frame):
         if self._received_payload or frame[5]:  # more
             self._received_payload.extend(frame[11])
         if not frame[5]:
```

Comment:
not frame[5] means the end of the message, the delivery tag is added at the first frame of the message, when we get larger messages this will be None